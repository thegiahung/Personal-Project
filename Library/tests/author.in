List author
author Harper Lee
author J.R.R. Tolkien
add collection books.csv
list authors
author Harper Lee
author J.R.R. Tolkien
author Me
author Her
exit
