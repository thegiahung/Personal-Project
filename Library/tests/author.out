user: user: No books in system.

user: No books in system.

user: 17 books successfully added.

user: Bob Kane
Douglas Adams
Harper Lee
Homer
J.K. Rowling
J.R.R. Tolkien
Jamie Lee Curtis
T.S. Eliot
Virgil
W.E. Johns

user: To Kill a Mockingbird (Harper Lee)

user: The Hobbit (J.R.R. Tolkien)

user: No books by Me.

user: No books by Her.

user: Ending Library process.
